packages/evasive-transform/src/location-unmapper.js

```diff
@@ -4,7 +4,7 @@
  * @module
  */
 
-import { SourceMapConsumer } from 'source-map';
+import { SourceMapConsumer } from 'source-map-js';
```

Comment:
I’d be happy to review this change to evasive-transform separately to make this one smaller and  land it faster.